src/plugins/rpd/rpdplugin.h

```diff
@@ -0,0 +1,86 @@
+#ifndef RPDPLUGIN_H
+#define RPDPLUGIN_H
```

Comment:
I'm no longer using include guards in this project, please just write `#pragma once`.

Reply:
done